apps/web/src/app/(dashboard)/admin/documents/[id]/page.tsx

```diff
@@ -81,6 +82,10 @@ export default async function AdminDocumentDetailsPage({ params }: AdminDocument
           ))}
         </Accordion>
       </div>
+
+      <hr className="my-4" />
+
+      {document && !document.deletedAt && <DeleteDocumentDialog document={document} />}
```

Comment:
We don't need to worry if the user has already deleted the document, we as admins would want the authoritative ability to delete any document. Otherwise a user may upload a malicious document and then delete it after hooking the target which stops us from removing it from the platform.